### ISSUE TITLE
system clipboard is lost when vim is put in background (or closed)

### ISSUE THREAD

**Issue by zkrx** [vim/vim#4841]
**Describe the bug**
When putting something into the system clipboard using e.g. "+yy, its content is lost once the vim instance is put in background using ctrl+z (or when vim is closed).

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Open any file using vim
2. "+yy one line
3. Ctrl+z (or quit vim)
4. try to paste it in any application (gedit, terminal, ...)
5. Oberve that it does not paste anything

If you skip step 3, you will see the pasted text in step 5 as expected.

**Expected behavior**
The system buffer should not be lost when the application is put in background (or closed).

**Environment (please complete the following information):**
$ vim --version
VIM - Vi IMproved 8.1 (2018 May 18, compiled Jul 29 2019 20:38:53)
Included patches: 1-1776
Compiled by Arch Linux
Huge version with GTK3 GUI.  Features included (+) or not (-):
+acl               -farsi             -mouse_sysmouse    -tag_any_white
+arabic            +file_in_path      +mouse_urxvt       +tcl/dyn
+autocmd           +find_in_path      +mouse_xterm       +termguicolors
+autochdir         +float             +multi_byte        +terminal
-autoservername    +folding           +multi_lang        +terminfo
+balloon_eval      -footer            -mzscheme          +termresponse
+balloon_eval_term +fork()            +netbeans_intg     +textobjects
+browse            +gettext           +num64             +textprop
++builtin_terms    -hangul_input      +packages          +timers
+byte_offset       +iconv             +path_extra        +title
+channel           +insert_expand     +perl/dyn          +toolbar
+cindent           +job               +persistent_undo   +user_commands
+clientserver      +jumplist          +postscript        +vartabs
+clipboard         +keymap            +printer           +vertsplit
+cmdline_compl     +lambda            +profile           +virtualedit
+cmdline_hist      +langmap           +python/dyn        +visual
+cmdline_info      +libcall           +python3/dyn       +visualextra
+comments          +linebreak         +quickfix          +viminfo
+conceal           +lispindent        +reltime           +vreplace
+cryptv            +listcmds          +rightleft         +wildignore
+cscope            +localmap          +ruby/dyn          +wildmenu
+cursorbind        +lua/dyn           +scrollbind        +windows
+cursorshape       +menu              +signs             +writebackup
+dialog_con_gui    +mksession         +smartindent       +X11
+diff              +modify_fname      +sound             -xfontset
+digraphs          +mouse             +spell             +xim
+dnd               +mouseshape        +startuptime       -xpm
-ebcdic            +mouse_dec         +statusline        +xsmp_interact
+emacs_tags        +mouse_gpm         -sun_workshop      +xterm_clipboard
+eval              -mouse_jsbterm     +syntax            -xterm_save
+ex_extra          +mouse_netterm     +tag_binary        
+extra_search      +mouse_sgr         -tag_old_static   

I'm using MATE on Arch Linux.

**Additional context**
May be relevant:
- https://freedesktop.org/wiki/ClipboardManager/
- https://wiki.ubuntu.com/ClipboardPersistence
- https://askubuntu.com/questions/12047/inconsistent-copy-and-paste-behaviour-is-there-a-fix

**Comment by chrisbra** [comment]
this has been just recently been discussed on the vim_use mailinglist:

https://groups.google.com/d/msg/vim_use/T9iMTBskDZk/9RpUj3ciAwAJ

TLDR: That is how the clipboard works. You need to install a clipboard manager for your desktop environment in order to make the clipboard work as you expect.
